### PR TITLE
Update/import export set props

### PR DESCRIPTION
**Pull request by mikejolley**
Expand should take care of some basic formatting for CSV specific items.

The importer however needs to take care of things which require ids or the post object e.g. image ids and attributes.

### CODE REVIEW

File: includes/import/class-wc-product-csv-importer.php [hunk 1]
```diff
@@ -549,10 +565,10 @@ protected function get_row_id( $parsed_data ) {
 			$row_data[] = $name;
 		}
 		if ( $id ) {
-			$row_data[] = __( 'ID: ', 'woocommerce' ) . $id;
+			$row_data[] = sprintf( __( 'ID %d', 'woocommerce' ), $id );
```
Comment (claudiosanches) [inline includes/import/class-wc-product-csv-importer.php]: I think you can keep `:`, at least should make easy to translate too.

**Review by claudiosanches — APPROVED** [review]
LGTM

**Comment by mikejolley** [comment]
Can tweak after feedback. Let's merge and give it another pass @claudiulodro

File: includes/import/abstract-wc-product-importer.php [hunk 2]
```diff
@@ -209,91 +170,107 @@ protected function get_product_object( $data ) {
...
-			$product->set_catalog_visibility( $data['catalog_visibility'] );
+	/**
+	 * Convert raw image URLs to IDs and set.
+	 *
+	 * @param WC_Product $product
+	 * @param array $data
+	 */
+	protected function set_image_data( &$product, $data ) {
+		// Image URLs need converting to IDs before inserting.
+		if ( isset( $data['image_id'] ) ) {
```
Comment (claudiulodro) [inline includes/import/abstract-wc-product-importer.php]: @mikejolley should this be `$data['raw_image_id']` and `$data['raw_gallery_image_ids']`? I wasn't able to find anything in here that converts from `$data['raw_image_id'] => $data['image_id']`.

**Pull request merged**